src/cli.ts

```diff
@@ -64,7 +64,7 @@ main(cliFlags, defaultConfig).catch((error) => {
 // Define Main Function
 
 async function main(args: typeof cliFlags, config: Config) {
-	setProcessAndTermTitle('md-to-pdf');
+	setProcessTitle('md-to-pdf');
```

Comment:
Can you just change this line to

```suggestion
	process.title = 'md-to-pdf';
```

and get rid of the helper and test?

No need to keep it if it's the only thing it does i think (: